src/content/docs/en/pages/deploy-journey/overview/overview.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Overview
+description: Azion Deploy products are designed to enhance the efficiency of managing edge resources in real time.
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_overview
+menu_namespace: deployMenu
+og_image: ''
+permalink: /documentation/products/deploy/overview/
+---
+
+import Button from '~/components/Button.astro'
+
+Azion Deploy products are designed to enhance the efficiency of managing edge resources in real time.  
+
+## Orchestrator
+
+Azion Edge Orchestrator is a versatile cloud management and provisioning service, specifically engineered for edge infrastructures and applications. It empowers operators and customers to manage and control edge resources in real-time, including the deployment, updating, and management of various components such as:
+
+- [Edge Services](/en/documentation/products/deploy/overview/#edge-services)
+- [Edge Nodes](/en/documentation/products/deploy/overview/#edge-nodes)
+
+The Orchestrator is adaptable to various environments, capable of running natively on the operating system, within containers, or on top of the [Azion Edge Runtime]().
+
+![Overview deploy](/assets/docs/images/uploads/deploy-draft.png)
+
+An Azion Edge Orchestrator agent is installed on the edge nodes and provides end-to-end encrypted remote node management from Real-Time Manager.
```

Comment:
pra usar o "the" antes de "edge nodes" tem que ter algum complemento. the edge nodes of Azion network, por exemplos